readme.md

```diff
@@ -175,11 +175,11 @@ How to Use
 To use, add the following to your `build.sbt`:
 
 ```scala
-libraryDependencies += "com.lihaoyi" %% "acyclic" % "0.1.7" % "provided"
+libraryDependencies += "com.lihaoyi" %% "acyclic" % "0.3.0" % "provided"
```

Comment:
I forgot my sbt foo too, but in scoverage the dependencies have an exrta `cross (CrossVersion.full)`. Probably something like this (haven't tested it):

```suggestion
libraryDependencies += "com.lihaoyi" %% "acyclic" % "0.3.0" cross (CrossVersion.full) % "provided"
```